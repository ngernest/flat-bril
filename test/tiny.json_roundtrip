{
  "functions": [
    {
      "args": [],
      "instrs": [
        {
          "dest": "v",
          "op": "const",
          "type": "int",
          "value": 5
        },
        {
          "args": [
            "v"
          ],
          "funcs": [],
          "labels": [],
          "op": "print"
        }
      ],
      "name": "main"
    }
  ]
}
