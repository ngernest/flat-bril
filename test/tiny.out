@main
	op: const		dest: v 		type: int		value: 5	
	op: print		args: ["v"]	
